src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1704,6 +1704,10 @@ static bool addTriggeredSubscriptions_noCache
           mqttInfo,
           aList, "", "", covered);
 
+      // Release after using them to fill the TriggeredSubscription
+      httpInfo.release();
+      mqttInfo.release();
+
```

Comment:
Even if the method `HttpInfo::release() ` checks if null, 

https://github.com/telefonicaid/fiware-orion/blob/ae5eb8552a12c730ec808642e5f06efc045f9851/src/lib/apiTypesV2/HttpInfo.cpp#L263-L278

Should be controlled if it was initialised?

```suggestion
      // Release after using them to fill the TriggeredSubscription
      if (sub.hasField(CSUB_MQTTTOPIC))
      {
        mqttInfo.release();
      }
      else
      {
        httpInfo.release();
      }
```